pyxem/signals/differential_phase_contrast.py

```diff
@@ -271,6 +271,135 @@ def get_magnitude_signal(self, autolim=True, autolim_sigma=4):
         pst._copy_signal2d_axes_manager_metadata(self, signal)
         return signal
 
+    def phase_retrieval(self, method='kottler', mirroring=False,
+                        mirror_flip=False):
+        """Retrieve the phase from two orthogonal phase gradients.
+
+        The formulae taken for different methods are from the following refs:
+
+        'kottler' --- Equation 4
+        C. Kottler, C. David, F. Pfeiffer, and O. Bunk, "A two-directional
+        approach for grating based differential phase contrast imaging using
+        hard x-rays," Opt. Express 15, 1175-1181, 2007
+
+        'arnison' --- Equation 6
+        Arnison MR, Larkin KG, Sheppard CJ, Smith NI, Cogswell CJ. Linear
+        phase imaging using differential interference contrast microscopy.
+        J Microsc. 2004 Apr;214(Pt 1):7-12.
+
+        'frankot' --- Equation 21
+        R. T. Frankot and R. Chellappa, "A method for enforcing integrability
+        in shape from shading algorithms," in IEEE Transactions on Pattern
+        Analysis and Machine Intelligence, vol. 10, no. 4, pp. 439-451, 1988
+
```

Comment:
Can the references be dropped into a reference section at the bottom inline with (https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)

Reply:
Moved to the reference section and also used the same ref style